src/types/index.ts

```diff
@@ -311,6 +311,10 @@ export interface CallOptions {
     nonce?: Numeric;
     maxFee?: Numeric;
     rawOutput?: boolean;
+    token?: string;
```

Comment:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/293/commits/c75e81f738dd0918f72e603e96c032864d00ddfa My thought is it could have an effect on the fee estimation when passed since it is also passed on the declare command.